index.html

```diff
@@ -1980,30 +1974,26 @@ <h2>The <dfn>record type</dfn> string</h2>
 </section> <!-- Data types and content -->
 
 
-<section> <h2>The NDEFReader and NDEFWriter objects</h2>
-  The objects provide a way for the <a>browsing context</a> to
-  use NFC functionality.
-  They allow for writing <a>NDEF message</a>s to <a>NFC tag</a>s within range,
-  and to act on incoming <a>NDEF message</a>s from <a>NFC tag</a>s.
+<section> <h2>The NDEFReader object</h2>
+  <p>
+    The <dfn>NDEFReader</dfn> is an object that exposes NFC functionality to the
+    <a>browsing context</a>: reading data when a device, such as a tag, is within
```

Comment:
I believe we should explicitly mention NDEF Message for reading like we do for writing

Reply:
Done.